lib/sandbox/execute.js

```diff
@@ -214,7 +216,18 @@ module.exports = function (bridge, glob) {
                     var eventId = timers.setEvent(callback);
 
                     bridge.dispatch(executionRequestEventName, options.cursor, id, eventId, request);
-                }, dispatchAssertions, new PostmanCookieStore(id, bridge, timers), {
+                },
+                /* onSkipRequest = */ () => {
+                    // throw an error if skipRequest is called from test script
+                    // as we only want to provide this api in prerequest script
+                    if (execution && execution.target === 'test') {
+                        throw new ReferenceError('skipRequest is not defined');
```

Comment:
It should be a TypeError since pm.execution exists in tests script. `TypeError: pm.execution.skipRequest is not a function`

Reply:
Addressed